tests/h/views/api_profile_test.py

```diff
@@ -0,0 +1,10 @@
+import mock
+
+from h.views import api_profile
+
+
+@mock.patch('h.session.profile')
```

Comment:
This is fine, just in case you didn't come across it yet, in recent months we opted for explicitely creating fixtures, with one of the advantages being you can give it a name yourself:

```python
@pytest.fixture
def profile(patch):
    return patch('h.session.profile')
```

Note: This also uses the `patch` fixture that is defined in `tests/h/conftest.py`.